scripts/cc-cli/src/commands/bond.ts

```diff
@@ -0,0 +1,64 @@
+import { Command, OptionValues } from "commander";
+import { newApi } from "../api";
+import { getSeedFromOptions } from "../utils/account";
+import { bond, parseRewardDestination } from "../utils/bond";
+import { promptContinue } from "../utils/promptContinue";
+
+export function makeBondCommand() {
+  const cmd = new Command("bond");
+  cmd.description("Bond CTC from a Stash account");
+  cmd.option("-a, --amount [amount]", "Amount to bond");
+  cmd.option("-s, --seed [seed]", "Specify mnemonic phrase to bond from");
```

Comment:
For the sake of consistency

```suggestion
  cmd.option("-s, --seed [mnemonic]", "Specify mnemonic phrase to bond from");
```